tools/wake-format/emitter.cpp

```diff
@@ -392,6 +464,16 @@ wcl::doc Emitter::walk_case(ctx_t ctx, CSTElement node) {
   MEMO(ctx, node);
   assert(node.id() == CST_CASE);
 
+  if (traits[node].format_off) {
```

Comment:
hmm can we eliminate this somehow? I don't want to repeat this in every walk function. Seems like something walk_node can do by itself right? The sub walkers that walk_node dispatches to don't need these right? Or do we dispatch to these directly?

Either way I want to think about how to eliminate these